### ISSUE TITLE
Keyboard shortcuts not respecting OS layout configuration

### ISSUE THREAD

**Issue by danpalmer** [ForkIssues/Tracker#815]
I use the "Dvorak Command+QWERTY" layout for macOS. This is the Dvorak layout for regular typing, but when CMD is held, it switches to the "QWERTY" layout for keyboard shortcuts as these are typically more about location than the actual letter. This is a standard OS system preference, no custom tools involved.

With a commit highlighted, the shortcut to copy the commit hash is "Cmd+C". However, with this keyboard layout, Cmd+C doesn't work, and instead Cmd+I does this action, implying that Fork is not respecting the system configuration correctly.

**Comment by AdamClements** [comment]
I'm having this same problem, couldn't work out why fixup etc wasn't working in the rebase dialog, turns out it's fine if I switch to QWERTY

**Comment by jchatard** [comment]
Yes same here, I'm using a french alternative to AZERTY, Optimot.fr, but all shorcuts respond to Qwerty regardless of current OS keyboard layout.
